dig.go

```diff
@@ -131,6 +131,12 @@ type Container struct {
 
 	// Source of randomness.
 	rand *rand.Rand
+
+	// Ensure verified acyclic before Invoke
+	verifiedAcyclic bool
```

Comment:
nit: Can you rename this to `isAcyclic` or `isVerifiedAcyclic`?  I misread
this as `verifyCyclic`, which made it confusing next to the
`deferAcyclicVerification` option.